modules/alloy/src/main/ts/ephox/alloy/positioning/mode/SelectionAnchor.ts

```diff
@@ -15,14 +15,21 @@ import * as ContentAnchorCommon from './ContentAnchorCommon';
 const descendOnce = (element: SugarElement<Node>, offset: number): Descend.ElementAndOffset<Node> =>
   SugarNode.isText(element) ? Descend.point(element, offset) : Descend.descendOnce(element, offset);
 
-const getAnchorSelection = (win: Window, anchorInfo: SelectionAnchor): Optional<SimRange> => {
+const isSimRange = (detail: SimRange | SugarElement<Node>[]): detail is SimRange =>
+  (detail as SimRange).start !== undefined;
```

Comment:
`start` seems a bit too generic to use for determining the shape of the object. I suggest using `foffset` as that's highly unlikely to be used anywhere else.
```suggestion
  (detail as SimRange).foffset !== undefined;
```